### DISCUSSION TITLE
Support additional ad-hoc filters

### DISCUSSION THREAD

**Discussion by RoryCrispin** [grafana/clickhouse-datasource#406] [Ideas]
Ad-hoc filter variables can be added to a dashboard and used to filter the ClickHouse data source results. 
The ad-hoc filter UI offers 
```
=
!=
>
<
=~
!~
```
But the CH datasource does not support the regex expressions `~=` and `!~`
Either, the UI should hide these fields, or support for them should be added. 

I'm not sure if the plugin has support for hiding the options, however.

**Comment by mshustov** [comment]
> ~= and !~

It's a bug. Created a bug fix https://github.com/grafana/clickhouse-datasource/pull/414
